crashed after signing up a logining in

**Describe the bug**
crashed after signing up a logining in
was working yesterday

**To Reproduce**
updated lens 5
was forced to create an account cause the 'skip' button didn't show
logged in
tried accessing my gcp cluster
crashed
'App crash at /overview'

Component stack:


    at Icon (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:582595:9)
    at _a (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:594495:17)
    at p
    at a
    at http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:454338:23
    at http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:454384:31
    at http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:454445:30
    at div
    at NonInjectedSidebarItem (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/chunks/888.js:571:9)
    at div
    at div
    at NonInjectedSidebar (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/chunks/888.js:780:26)
    at div
    at div
    at NonInjectedMainLayout (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:586079:9)
    at ErrorBoundary (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:582243:9)
    at NonInjectedClusterFrame (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/chunks/888.js:2924:9)
    at ThemeProvider (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:127160:24)
    at Router (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:454780:30)
Error stack:

Error: Cannot find module './<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 23.0.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 994.3 711.4" style="enable-background:new 0 0 994.3 711.4;" xml:space="preserve">
<style type="text/css">
	.st0{fill-rule:evenodd;clip-rule:evenodd;}
</style>
<g>
	<path class="st0" d="M342.6,402.4c30.3,0,54.9-24.5,54.9-54.8c0-30.3-24.6-54.8-54.9-54.8c-30.3,0-54.8,24.5-54.8,54.8
		C287.8,377.9,312.3,402.4,342.6,402.4z"/>
	<path class="st0" d="M342.6,429.3c22.6,0,43-9.1,57.8-23.9c14.8-14.8,23.9-35.2,23.9-57.8c0-22.5-9.2-43-23.9-57.8
		c-14.8-14.8-35.2-23.9-57.8-23.9c-22.5,0-43,9.1-57.7,23.9c-14.8,14.8-23.9,35.2-23.9,57.8c0,22.5,9.1,43,23.9,57.8
		C299.6,420.2,320,429.3,342.6,429.3z M292.3,297.3c12.9-12.9,30.7-20.8,50.3-20.8c19.7,0,37.4,7.9,50.3,20.8
		c12.9,12.9,20.8,30.7,20.8,50.3c0,19.6-8,37.4-20.8,50.3c-12.9,12.9-30.6,20.8-50.3,20.8c-19.6,0-37.4-7.9-50.3-20.8
		c-12.9-12.9-20.8-30.7-20.8-50.3C271.5,328,279.4,310.2,292.3,297.3z"/>
	<path class="st0" d="M651.7,292.8c-30.3,0-54.8,24.5-54.8,54.8c0,30.3,24.5,54.8,54.8,54.8c30.3,0,54.8-24.5,54.8-54.8
		C706.5,317.3,682,292.8,651.7,292.8z"/>
	<path class="st0" d="M651.7,265.9c-22.6,0-43,9.1-57.8,23.9c-14.8,14.8-23.9,35.2-23.9,57.8c0,22.5,9.2,43,23.9,57.8
		c14.8,14.8,35.2,23.9,57.8,23.9c22.5,0,43-9.1,57.7-23.9c14.8-14.8,23.9-35.2,23.9-57.8c0-22.5-9.1-43-23.9-57.8
		C694.7,275,674.3,265.9,651.7,265.9z M702,397.9c-12.9,12.9-30.6,20.8-50.3,20.8c-19.7,0-37.4-7.9-50.3-20.8
		c-12.9-12.9-20.8-30.7-20.8-50.3c0-19.6,8-37.4,20.8-50.3c12.9-12.9,30.6-20.8,50.3-20.8c19.6,0,37.4,7.9,50.3,20.8
		c12.9,12.9,20.8,30.7,20.8,50.3C722.8,367.2,714.9,385,702,397.9z"/>
	<path class="st0" d="M993.3,40c-18.8,31.1-53.4,81.8-60.7,95.1l-4.9-57.9l-95.2,100.5c-32.7-62.1-92.5-107.9-163.4-121.4
		C651.3,24.6,579.3,1.8,505.3,0.1C502.6,0,499.9,0,497.1,0c-2.7,0-5.5,0-8.2,0.1c-74,1.7-146,24.5-163.8,56.2
		c-71,13.5-130.8,59.3-163.4,121.4L66.5,77.2l-4.9,57.9C54.3,121.7,19.8,71.1,1,40c-5.1,18.4,11.3,161,15.8,179.4
		c4.4,18.1,34.2,142.5,57,140.4c3.2-1.2,9.3-26.3,16.9-48.8l52.2,97.1c24.3,97,103,172.9,201.4,193.3c-2.1,12-2.2,24.3-1.4,34.9
		c2.1,26.4,12.5,68.5,43.2,74.5c5.3,1,10.6,0.8,15.8-0.6c19.9-5.3,42.3-16.5,62.5-28.9c5.2,2.4,10.9,3.9,17,3.9H513
		c6.1,0,11.8-1.4,17-3.8c20.1,12.4,42.6,23.5,62.4,28.9c5.2,1.4,10.5,1.6,15.8,0.6c30.7-6,41.1-48.1,43.2-74.5
		c0.8-10.5,0.7-22.9-1.4-34.9c98.4-20.4,177.1-96.3,201.4-193.3l52.2-97.1c7.6,22.5,13.8,47.5,16.9,48.8
		c22.8,2.1,52.6-122.3,57-140.4C982,201,998.4,58.3,993.3,40z M497.1,31c79.3,0.7,161.1,24.7,154.5,42.6l-55.7,149.9
		c-4.1,10.9-9.6,21.2-21.2,21.2h-77.5h-77.5c-11.7,0-17.2-10.3-21.2-21.2L342.6,73.5C336,55.7,417.8,31.6,497.1,31z M135,285.7v58.9
		c0,3.4,0.1,6.7,0.2,10.1l-56.9-96L69,320C41.5,240.6,27.1,180.7,15.8,95.5l58.3,106.7l10.7-80.7l67.2,76.9
		C141.1,225.4,135,254.9,135,285.7z M396.4,689.7c-37.4,10-51-133.8,3.5-112.9c12,4.6,27.6,13,46.4,23.7c-3.2,5.7-5,12.3-5,19.3v2.6
		c-8.9-2.8-21-5.8-36.5-9.4c25.7,16.2,17.9,12,35.2,20.6c-19.2,11.4-10.5,5.7-37.2,25.8c16.3-5.4,29.1-9.8,38.5-13.6
		c0.1,7.6,2.4,14.7,6.2,20.7C432.2,675.6,413.7,685.1,396.4,689.7z M597.9,689.7c-17.2-4.6-35.8-14.1-51.2-23.2
		c3.6-5.7,5.8-12.4,6.2-19.5c9,3.5,20.9,7.6,35.5,12.4c-25.9-19.4-18.6-14.8-35.4-24.7v-2c15-7.4,8.7-4.1,33.4-19.7
		c-13.8,3.2-24.9,6-33.4,8.5v-1.7c0-7-1.8-13.6-5-19.3c18.9-10.7,34.4-19.1,46.4-23.7C648.8,555.9,635.3,699.7,597.9,689.7z
		 M838,344.6C838,461.2,754.2,559,643.8,581c-4.5-9.8-11.2-18-20.9-23c-11.5-6-24.2-5.5-36.1-0.9c-12.5,4.8-25,11.5-36.7,18
		c-5.9,3.2-11.8,6.6-17.6,9.9c-5.8-3.3-12.5-5.2-19.6-5.2h-31.6c-7.1,0-13.8,1.9-19.6,5.2c-5.8-3.4-11.7-6.7-17.6-9.9
		c-11.7-6.5-24.1-13.2-36.7-18c-11.9-4.6-24.6-5-36.1,0.9c-9.7,5-16.4,13.3-20.9,23c-110.3-22-194.2-119.8-194.2-236.3v-58.9
		c0-100.7,71.1-185.6,165.7-207c0.2,0.8,0.4,1.5,0.7,2.2l55.7,149.9c7,18.8,18.8,35.1,41.1,35.1h77.7h77.7
		c22.3,0,34.1-16.3,41.1-35.1l55.7-149.9c0.3-0.7,0.5-1.5,0.7-2.2c94.5,21.4,165.7,106.3,165.7,207V344.6z M925.3,320l-9.4-61.3
		l-56.9,96c0.1-3.4,0.2-6.7,0.2-10.1v-58.9c0-30.8-6.1-60.3-17.1-87.3l67.2-76.9l10.7,80.7l58.3-106.7
		C967.2,180.7,952.8,240.6,925.3,320z"/>
	<path class="st0" d="M595.8,521.3h-98.6h-98.6c-70.4,0-107.9-34.6-120.7-49.2l-10.5,8.8c14.5,16.5,55.7,54,131.2,54h98.6h98.6
		c75.4,0,116.7-37.5,131.2-54l-10.5-8.8C703.7,486.7,666.1,521.3,595.8,521.3z"/>
</g>
</svg>
.svg'
    at webpackContextResolve (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:615307:11)
    at webpackContext (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:615302:11)
    at Icon.render (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:582651:100)
    at qi (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:453323:188)
    at pi (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:453322:173)
    at ck (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:453405:427)
    at bk (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:453386:347)
    at ak (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:453386:278)
    at Tj (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:453386:138)
    at Lj (http://3fd37ed679b6191bda5b3e90cb182af8.localhost:50602/build/Lens.js?700ce8d46c05fdd6c192:453379:163)

--- a/src/renderer/components/icon/icon.tsx
+++ b/src/renderer/components/icon/icon.tsx
@@ -71,7 +71,28 @@ export interface BaseIconProps {
   material?: string;
 
   /**
-   * Either an SVG data URL or one of the following strings
+   * Either an SVG XML or one of the following names
+   * - configuration
+   * - crane
+   * - group
+   * - helm
+   * - install
+   * - kube
+   * - lens-logo
+   * - license
+   * - logo-lens
+   * - logout
+   * - nodes
+   * - push_off
+   * - push_pin
+   * - spinner
+   * - ssh
+   * - storage
+   * - terminal
+   * - user
+   * - users
+   * - wheel
+   * - workloads
    */
   svg?: string;
 

--- a/src/renderer/components/icon/icon.tsx
+++ b/src/renderer/components/icon/icon.tsx
@@ -12,7 +12,57 @@ import type { LocationDescriptor } from "history";
 import { cssNames } from "../../utils";
 import { withTooltip } from "../tooltip";
 import isNumber from "lodash/isNumber";
-import { decode } from "../../../common/utils/base64";
+import Configuration from "./configuration.svg";
+import Crane from "./crane.svg";
+import Group from "./group.svg";
+import Helm from "./helm.svg";
+import Install from "./install.svg";
+import Kube from "./kube.svg";
+import LensLogo from "./lens-logo.svg";
+import License from "./license.svg";
+import LogoLens from "./logo-lens.svg";
+import Logout from "./logout.svg";
+import Nodes from "./nodes.svg";
+import PushOff from "./push_off.svg";
+import PushPin from "./push_pin.svg";
+import Spinner from "./spinner.svg";
+import Ssh from "./ssh.svg";
+import Storage from "./storage.svg";
+import Terminal from "./terminal.svg";
+import User from "./user.svg";
+import Users from "./users.svg";
+import Wheel from "./wheel.svg";
+import Workloads from "./workloads.svg";
+
+/**
+ * Mapping between the local file names and the svgs
+ *
+ * Because we only really want a fixed list of bundled icons, this is safer so that consumers of
+ * `<Icon>` cannot pass in a `../some/path`.
+ */
+const localSvgIcons = new Map([
+  ["configuration", Configuration],
+  ["crane", Crane],
+  ["group", Group],
+  ["helm", Helm],
+  ["install", Install],
+  ["kube", Kube],
+  ["lens-logo", LensLogo],
+  ["license", License],
+  ["logo-lens", LogoLens],
+  ["logout", Logout],
+  ["nodes", Nodes],
+  ["push_off", PushOff],
+  ["push_pin", PushPin],
+  ["spinner", Spinner],
+  ["ssh", Ssh],
+  ["storage", Storage],
+  ["terminal", Terminal],
+  ["user", User],
+  ["users", Users],
+  ["wheel", Wheel],
+  ["workloads", Workloads],
+]);
 
 export interface BaseIconProps {
    /**
@@ -78,8 +128,8 @@ export interface BaseIconProps {
 export interface IconProps extends React.HTMLAttributes<any>, BaseIconProps {}
 
 export function isSvg(content: string): boolean {
-  // data-url for raw svg-icon
-  return String(content).includes("svg+xml");
+  // source code of the asset
+  return String(content).includes("<svg");
 }
 
 const RawIcon = withTooltip((props: IconProps) => {
@@ -131,13 +181,16 @@ const RawIcon = withTooltip((props: IconProps) => {
 
   // render as inline svg-icon
   if (typeof svg === "string") {
-    const dataUrlPrefix = "data:image/svg+xml;base64,";
-    const svgIconDataUrl = svg.startsWith(dataUrlPrefix) ? svg : require(`./${svg}.svg`);
-    const svgIconText = typeof svgIconDataUrl == "string" // decode xml from data-url
-      ? decode(svgIconDataUrl.replace(dataUrlPrefix, ""))
-      : "";
+    const svgIconText = isSvg(svg)
+      ? svg
+      : localSvgIcons.get(svg) ?? "";
 
-    iconContent = <span className="icon" dangerouslySetInnerHTML={{ __html: svgIconText }} />;
+    iconContent = (
+      <span
+        className="icon"
+        dangerouslySetInnerHTML={{ __html: svgIconText }}
+      />
+    );
   }
 
   // render as material-icon

--- a/types/mocks.d.ts
+++ b/types/mocks.d.ts
@@ -25,7 +25,11 @@ declare module "*.scss" {
 
 // Declare everything what's bundled as webpack's type="asset/resource"
 // Should be mocked for tests support in jestConfig.moduleNameMapper (currently in "/package.json")
-declare module "*.svg";
+declare module "*.svg" {
+  const content: string;
+  export = content;
+}
+
 declare module "*.jpg";
 declare module "*.png";
 declare module "*.eot";

--- a/webpack/renderer.ts
+++ b/webpack/renderer.ts
@@ -132,7 +132,7 @@ export function iconsAndImagesWebpackRules(): webpack.RuleSetRule[] {
   return [
     {
       test: /\.svg$/,
-      type: "asset/inline", // data:image/svg+xml;base64,...
+      type: "asset/source", // exports the source code of the asset, so we get XML
     },
     {
       test: /\.(jpg|png|ico)$/,

